security/libaes_siv/Portfile

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+# NOTE: Doesn't work with cmake 1.1 PortGroup
```

Comment:
Why doesn't it work with 1.1?

Reply:
Indeed, the plan is to migrate all ports to it. Maybe it simply needs:

`cmake.out_of_source no`

Reply:
That could well be.  It did look like the problem is that the upstream code doesn't like to build out of source.  I'm not convinced that making this the default in 1.1 was a good idea, given that it creates exactly this sort of incompatibility.

Given the amount of testing involved if I change anything, and the fact that a 1.0.1 version is expected to release soon, is it OK to leave it as is for now?

Reply:
It is not an incompatibility. That is just the default which can be easily changed. Most cmake ports use "yes" and hence why the default was changed. Using the old portgroup is not really on option because that is taking a step backward.

Reply:
I made the change (and retested on all 14 platforms) in the interests of avoiding further arguments, though it's a near certainty that there will be a new version of this port before all 253 of the other ports using cmake 1.0 have been updated. :-)